nuxeo-core/nuxeo-core-management-test/src/test/java/org/nuxeo/ecm/core/management/test/probes/TestProbes.java

```diff
@@ -266,4 +276,27 @@ public void testRepositoryStatusProbe() throws IOException {
         assertTrue(probeInfo.getStatus().isSuccess());
         assertEquals("{\"repositoryStatus\":\"ok\"}", result.toJson());
     }
+
+    @Test
+    public void testProbe() throws Exception {
```

Comment:
Why isn't the test in `TestStreamService` like in master?

Reply:
Because the code was refactored between 10.10 and master => the Probe classes are still in core modules